package.json

```diff
@@ -69,7 +69,7 @@
 		"build": "ttsc -p tsconfig.build.json",
 		"watch": "ttsc -w",
 		"lint": "eslint ./src --ext .ts",
-		"bindings": "./scripts/genbindings.sh deps/erdstall-contracts",
+		"bindings": "./scripts/genbindings.sh",
```

Comment:
hmm, the commit message says the opposite of what's happening here, no? Also, you don't update the actual script file in this commit. So it didn't need this argument in the first place?

Reply:
Ahh now I get it. Could you add a small sentence to the README on how to generate new bindings?